apps/meteor/client/lib/rooms/roomTypes/voip.ts

```diff
@@ -24,6 +24,9 @@ roomCoordinator.add(VoipRoomType, {
 	},
 
 	findRoom(identifier) {
+		if (!hasPermission('view-l-room') || !hasPermission('view-v-room')) {
+			return;
+		}
```

Comment:
```suggestion
```

It's better not to have this test at all. It'll only cause the room data to be loaded from the server every time.